Added a related PMML serialization project

vruusmann opened the pull request:
Opened a new pull request regarding documentation changes as suggested in PR #5151

vruusmann committed:
Added a related PMML serialization project

Opened a new pull request regarding documentation changes as suggested in PR #5151

agramfort commented on doc/related_projects.rst:
```diff
@@ -27,6 +27,9 @@ enhance the functionality of scikit-learn's estimators.
   Serialization of (some) scikit-learn estimators into PMML.
 
+- `sklearn2pmml <https://github.com/jpmml/sklearn2pmml>`_
+  Serialization of scikit-learn estimators and transformers in PMML data format.
```
LGTM but how does it compare to sklearn-pmml?

vruusmann replied:
The `sklearn2pmml` project can fully supports more than 20 estimator types (most popular clustering, classification amd regression types) and close to 10 transformer types. As such, it is *actually useful* for someone who is trying to migrate their SkLearn workflows to PMML.

The `sklearn-pmml` project, on the other hand, claims to support three estimator types (with limitations).

agramfort replied:
ok can you just write:

Serialization of more than 30 scikit-learn estimators and transformers

vruusmann replied:
I don't want to limit the description to *only* 30 supported features. Perhaps "a wide variety" would be more appropriate?

agramfort replied:
I said “more than" but whatever.

vruusmann committed:
Distinguished more clearly between two PMML serialization projects

amueller commented on doc/related_projects.rst:
```diff
@@ -27,6 +27,10 @@ enhance the functionality of scikit-learn's estimators.
   Serialization of (some) scikit-learn estimators into PMML.
 
+- `sklearn2pmml <https://github.com/jpmml/sklearn2pmml>`_
+  Serialization of a wide variety of scikit-learn estimators and transformers
```
I like the "more than 30" better but whatever. LGTM

amueller commented:
Should we add the main library https://github.com/jpmml/jpmml-sklearn ?
sklearn2pmml is just a thin wrapper....

vruusmann commented:
Sure, it may be a good idea to emphasize that `sklearn-pmml` is pure a Python library, whereas `sklearn2pmml` is a thin Python wrapper around a Java library.

"Serialization of a wide variety of scikit-learn estimators and transformers into PMML data format with the help of a [JPMML-SkLearn] (https://github.com/jpmml/jpmml-sklearn) library"?

amueller commented:
sounds good.

vruusmann committed:
Explained implementation details

The conversion functionality of the `sklearn2pmml` library is provided by a standalone Java library. It does not make much difference from the end user perspective. However, it makes all the difference from the development perspective.

agramfort commented:
thx
